git-hub

```diff
@@ -1471,6 +1474,143 @@ class PullCmd (IssueCmd):
             git_quiet(1, 'fetch', remote_url, remote_branch)
             git_quiet(1, 'checkout', 'FETCH_HEAD', *args.args)
 
+    class ReviewCmd (PullUtil):
+        cmd_help = "review the pull request"
+        @classmethod
+        def setup_parser(cls, parser):
+            parser.add_argument('pull',
+                help="number identifying the pull request to review")
+            parser.add_argument('-b', '--base', metavar='BASE',
+                default='upstream/master',
+                help="branch (or git ref) from which to start reviewing "
+                    "(defaults to 'upstream/master')")
+            parser.add_argument('-s', '--auto-skip', metavar='N', type=int,
+                help="automatically skip the first N commits")
+        @classmethod
+        def run(cls, parser, args):
+            # First confirm that a diff tool has been configured
+            if not git_config('diff.tool', prefix=''):
+                sys.stdout.write("Use '\033[94m"
```

Comment:
Also, please stick to using the `debugf()` `infof()`, `warnf()`, `errf()` and `die()` when possible, specially if you want to use colours. `sys.stdout` and `sys.stderr` should be virtually never used. If you need to use colors it would be nice to abstract it away also, like those functions or `ask()`. Having all the escape chars everywhere makes messages more complicated to handle (specially if at some point the tool is internationalized).